### PR TITLE
Filling in Use Cases Document

### PR DESCRIPTION
**Pull request by eric-schuh**
To this point we have been focusing on the Focal Use Cases, and the sequence diagrams specifically. This is a first pass at the rest of this use cases document. It is largely based off of the VC Use Cases document with updates to reflect the split in roles for the Issuer and Verifier, as discussed in the VC-API group and updated use case descriptions to highlight this bifurcation in roles beyond what was discussed in the VCWG.

***
<a href="https://pr-preview.s3.amazonaws.com/eric-schuh/vc-api-use-cases/pull/24.html" title="Last updated on Jan 17, 2023, 8:45 PM UTC (68ee885)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c-ccg/vc-api-use-cases/24/9c58bc5...eric-schuh:68ee885.html" title="Last updated on Jan 17, 2023, 8:45 PM UTC (68ee885)">Diff</a>

### CODE REVIEW

**Comment by jandrieu** [comment]
We should figure out how to clean up the ReSpec errors. The current respec preview rendering attempts (and fails) to link terms and other niceties, but currently generates errors, displayed with a squiggly red underline.

**Comment by eric-schuh** [comment]
In the User Roles diagram, revert back to the VC Use Cases roles but have the split in verifier and issuer and "components".  This had a lot of support in Dec-12 call. Lets split the roles and components to make the mental model cleaner. A component is used to actualize a role, the VC-API needs to talk about components in addition to the roles

**Comment by eric-schuh** [comment]
From Dec-12 Call: 
Ted - Does the "subject" belong as a role? Should the "holder" be split into a couple of different roles (issued to, or presenting to)? i.e. "Presenter" and "Issuee"

Manu - Three roles seems good, don't want to add complexity to the roles.

**Comment by eric-schuh** [comment]
Remove capitalization from terms

**Comment by eric-schuh** [comment]
Consider an "existing use" section where we can put real world deployments (see W3C CSV group for example of this type of section)

File: index.html [hunk 1]
```diff
@@ -152,45 +152,747 @@
...
   <section>
     <!-- Editor - @eric-schuh -->
-    <h2>Introduction</h2>
+    <h2>Abstract</h2>
       <p>
-        For Verifiable Credentials to be realized as a usable data format there needs to exist a neutral, 
-        standard-governed way for Issuers, Holders and Verifiers of these credentials to transport them between said entities.
-        TODO: Finish introduction
+        Fundamentally, the Verifiable Credential API (VC-API) is a standard developed to give technologists a way to enable holders of
+        VCs to use them how and where they choose. Verifiable Credentials make claims which are a qualification, achievement, quality, or piece
```
Comment (TallTed) [inline index.html]:
```suggestion
        VCs to use them how and where they choose. Verifiable Credentials make claims which may be a qualification, achievement, quality, or piece 
```

File: index.html [hunk 2]
```diff
@@ -152,45 +152,747 @@
...
-    <h2>Introduction</h2>
+    <h2>Abstract</h2>
       <p>
-        For Verifiable Credentials to be realized as a usable data format there needs to exist a neutral, 
-        standard-governed way for Issuers, Holders and Verifiers of these credentials to transport them between said entities.
-        TODO: Finish introduction
+        Fundamentally, the Verifiable Credential API (VC-API) is a standard developed to give technologists a way to enable holders of
+        VCs to use them how and where they choose. Verifiable Credentials make claims which are a qualification, achievement, quality, or piece 
+        of information about an entity's background such as a name, government ID, payment provider, home address, or university degree. 
+        The use cases outlined here are provided in order to make progress toward possible future standardization and interoperability of the transport
```
Comment (TallTed) [inline index.html]:
```suggestion
        The use cases outlined here are provided to help make progress toward possible future standardization and interoperability of the transport
```

File: index.html [hunk 3]
```diff
@@ -152,45 +152,747 @@
...
+        of both low- and high-stakes claims as outlined in the VC Use Cases. The use cases in this document focus on concrete 
+        scenarios that the technology defined by the group should address.
+      </p>
+  </section>
+  <section id='sotd'>
+      <p>
+        This document represents a concise but limited collection of use cases readers
+        should review alongside the <a href="https://w3c-ccg.github.io/vc-api/">Verifiable
```
Comment (TallTed) [inline index.html]:
```suggestion
        This document represents a concise but limited collection of use cases meant
        to be reviewed alongside the <a href="https://w3c-ccg.github.io/vc-api/">Verifiable
```

File: index.html [hunk 4]
```diff
@@ -152,45 +152,747 @@
...
+        Credentials API</a>.
+      </p>
+      <p>
+        The work on this document is ongoing and subject to change as the conversation around the
```
Comment (TallTed) [inline index.html]:
```suggestion
        Work on this document is ongoing and subject to change as the conversation around the
```

File: index.html [hunk 5]
```diff
@@ -152,45 +152,747 @@
...
+        VC API evolves in the main group. This should currently be treated as a draft document and
+        as such will have errors. As the VC API gets closer to a completed specification this 
+        document should reflect the current understanding and status of that work.
```
Comment (TallTed) [inline index.html]:
```suggestion
        as such will have errors. As the VC API gets closer to a completed specification, this 
        document should reflect the current understanding and status of that work. 
```

File: index.html [hunk 6]
```diff
@@ -152,45 +152,747 @@
...
+      <p>
+        The VC-API Working Group at the W3C is developing standards for
+        exchanging Verifiable Credentials that have been verified by a third
+        party and to make them easier for holders to use on the Web and in life.
     </p>
     <section>
-      <h3>Existing Work</h3>
+      <h3>Importance of this work</h3>
+        <p>
+          Entities (people, organizations, devices) need to make many kinds of
```
Comment (TallTed) [inline index.html]:
```suggestion
          <a>Entities</a> (people, organizations, devices) need to make many kinds of
```

File: index.html [hunk 7]
```diff
@@ -152,45 +152,747 @@
...
+        There are four roles supported by <a>Verifiable Credentials</a> as defined by the 
+        VCWG: <a>Issuer</a>,
+        <a>Verifier</a>, <a>Subject</a>, and <a>Holder</a>.
+      </p>  
+      <p>
+        The above roles were those defined for the VCWG, however, due to the way most 
+        production software on the internet works, for the purposes of the VC-API there arises
+        the need to delve into some detail as to how the Issuer and Verifier roles are structured.
+        This is due to the fact that there are many use cases where one company is making use of 
+        services to handle part of the technology stack for them. This has lead to both Verifier
```
Comment (TallTed) [inline index.html]:
```suggestion
        services to handle part of the technology stack. This has lead to both Verifier
```

File: index.html [hunk 8]
```diff
@@ -152,45 +152,747 @@
...
+        and Issuer splitting into two entities each: <a>Issuer Service</a>, <a>Issuer Coordinator</a>
```
Comment (TallTed) [inline index.html]:
```suggestion
        and Issuer splitting into two entities each: <a>Issuer Service</a>, <a>Issuer Coordinator</a>,
```

File: index.html [hunk 9]
```diff
@@ -152,45 +152,747 @@
...
+        <a>Verifier Service</a>, and <a>Verifier Coordinator</a>. 
+      </p>
+      <p>
+        This leads to the full set of roles being considered in the VC-API as:
```
Comment (TallTed) [inline index.html]:
```suggestion
        Thus, the full set of roles considered in the VC-API are
```

File: index.html [hunk 10]
```diff
@@ -152,45 +152,747 @@
...
+  
   <section>
-    <h2>Use Cases</h2>
-    <p>New cases to be aggregated here, from VC and DID Use Case documents.
-    </p>
+    <h2>User Roles</h2>
+      <p>
+        There are four roles supported by <a>Verifiable Credentials</a> as defined by the 
+        VCWG: <a>Issuer</a>,
+        <a>Verifier</a>, <a>Subject</a>, and <a>Holder</a>.
```
Comment (TallTed) [inline index.html]:
```suggestion
        <a>Verifier</a>, and <a>Holder</a>.
```

File: index.html [hunk 11]
```diff
@@ -152,45 +152,747 @@
...
+        the need to delve into some detail as to how the Issuer and Verifier roles are structured.
+        This is due to the fact that there are many use cases where one company is making use of 
+        services to handle part of the technology stack for them. This has lead to both Verifier
+        and Issuer splitting into two entities each: <a>Issuer Service</a>, <a>Issuer Coordinator</a>
+        <a>Verifier Service</a>, and <a>Verifier Coordinator</a>. 
+      </p>
+      <p>
+        This leads to the full set of roles being considered in the VC-API as: 
+        <a>Issuer Service</a>, <a>Issuer Coordinator</a>, <a>Verifier Service</a>, <a>Verifier Coordinator</a>,
+        <a>Subject</a>, and <a>Holder</a>.
```
Comment (TallTed) [inline index.html]:
```suggestion
        and <a>Holder</a>.
```

File: index.html [hunk 12]
```diff
@@ -152,45 +152,747 @@
...
+        </figcaption>
+      </figure>
+
+    <dl>
+      <dt>
+        Issuer Service
+      </dt>
+      <dd>
+        The <a>entity</a> that creates a <a>claim</a> and associates it with a
+        particular <a>subject</a>.
```
Comment (TallTed) [inline index.html]: I don't think this is correct. As I understand it, the Issuer Service (software) delivers a VC, which contains the claims made by the Issuer Coordinator (human, possibly plus software) about one or more subjects.

File: index.html [hunk 13]
```diff
@@ -152,45 +152,747 @@
...
+      </dt>
+      <dd>
+        The <a>entity</a> that deals with business rules and policies around a
+        user's request for a particlar <a>claim</a> to be issued.
+      </dd>
+      <dt>
+        Verifier Service
+      </dt>
+      <dd>
+        The <a>entity</a> performing the verifaction of a <a>claim</a> about a given <a>subject</a>.
```
Comment (TallTed) [inline index.html]:
```suggestion
        The <a>entity</a> performing the verification of a <a>Verifiable Credential</a> containing <a>claims</a> about a given <a>subject</a>.
```

File: index.html [hunk 14]
```diff
@@ -152,45 +152,747 @@
...
+      </dd>
+      <dt>
+        Verifier Coordinator
+      </dt>
+      <dd>
+        The <a>entity</a> that deals with business rules and policies around a
+        user's request for a particular <a>claim</a> to be verified.
```
Comment (TallTed) [inline index.html]:
```suggestion
        user's request for a particular <a>Verifiable Credential</a> to be verified. 
```

File: index.html [hunk 15]
```diff
@@ -152,45 +152,747 @@
...
+  
   <section>
-    <h2>Use Cases</h2>
-    <p>New cases to be aggregated here, from VC and DID Use Case documents.
-    </p>
+    <h2>User Roles</h2>
+      <p>
+        There are four roles supported by <a>Verifiable Credentials</a> as defined by the 
+        VCWG: <a>Issuer</a>,
+        <a>Verifier</a>, and <a>Holder</a>.
```
Comment (TallTed) [inline index.html]: Either the number changes as I have, or there's another role to name in the list...
```suggestion
        There are three roles supported by <a>Verifiable Credentials</a> as defined by the 
        VCWG: <a>Issuer</a>,
        <a>Verifier</a>, and <a>Holder</a>.
```

File: index.html (hunk 12 above)
Comment (eric-schuh) [reply index.html]: Updated language to reflect this, new language is in the following commit:

772a31769b8263ff00adbb591110c2e31c7ce198

**Comment by eric-schuh** [comment]
role-components
user-roles

New images may need to be changed later, can't figure out how to change the images in the PR files listing or add new ones.

**Comment by eric-schuh** [comment]
@jandrieu ready for review, I may have to add new images after this PR is accepted since I can't find a way to change the images in the PR itself. Once done I will take care of images.

Also I looked into fixing the respec but it is not clear what is wrong with the terms.html call. Everything looks identical to the vc-use-cases in the index.html when relating to terms so will need to dig into that more later.

File: index.html [hunk 16]
```diff
@@ -240,43 +240,54 @@ <h3>Use case model</h3>
...
     <h2>User Roles</h2>
       <p>
-        There are four roles supported by <a>Verifiable Credentials</a> as defined by the 
-        VCWG: <a>Issuer</a>,
-        <a>Verifier</a>, and <a>Holder</a>.
+        There are three roles supported by <a>Verifiable Credentials</a> as defined by the 
+        VCWG: <a>Issuer</a>, <a>Verifier</a>, and <a>Holder</a>. A fourth role is defined in
+        the VC Use Cases, this is the role of subject, however, for the purposes of the VC-API
+        the subject does not have an explicit role outside of acting as both Holder and Subject.
+        Do to this, the Subject role does not appear here in the VC-API.
```
Comment (TallTed) [inline index.html]:
```suggestion
        VCWG: <a>Issuer</a>, <a>Verifier</a>, and <a>Holder</a>. In the VC Use Cases, a fourth
        role — that of subject — is defined. However, for the purposes of the VC-API, the subject
        is not relevant except when the same entity is both the Holder and the subject. For this
        reason, the Subject role does not appear in the VC-API.
```

File: index.html [hunk 17]
```diff
@@ -240,43 +240,54 @@ <h3>Use case model</h3>
...
+        <figcaption>
+          VC-API User Roles
+        </figcaption>
+      </figure>
       <p>
         The above roles were those defined for the VCWG, however, due to the way most 
         production software on the internet works, for the purposes of the VC-API there arises
         the need to delve into some detail as to how the Issuer and Verifier roles are structured.
         This is due to the fact that there are many use cases where one company is making use of 
         services to handle part of the technology stack. This has lead to both Verifier
```
Comment (TallTed) [inline index.html]:
```suggestion
        services to handle part of the technology stack. This has led to both Verifier
```

File: index.html [hunk 18]
```diff
@@ -240,43 +240,54 @@ <h3>Use case model</h3>
...
            style="display:block; margin:auto"
-           src="user-roles.png" />
+           src="role-components.png" />
         <figcaption>
-          VC-API User Roles
+          VC-API Role Components
         </figcaption>
       </figure>
+      <p>
+        The combination of the respective Service and Coordinator components combine to fulfill
```
Comment (TallTed) [inline index.html]:
```suggestion
        The respective Service and Coordinator components combine to fulfill
```

File: index.html [hunk 19]
```diff
@@ -240,43 +240,54 @@ <h3>Use case model</h3>
...
+        the role of the Issuer and Verifier as described in the VCWG. 
+        It is fully expected by the VC-API that software will exist that may fulfil only part of
+        each of the roles by implimenting any combnation of these components:
```
Comment (TallTed) [inline index.html]:
```suggestion
        It is fully expected by the VC-API that software will exist that may fulfill only part of
        any of the roles, by implementing any combination of these component roles: 
```

File: index.html [hunk 20]
```diff
@@ -240,43 +240,54 @@ <h3>Use case model</h3>
...
 
     <dl>
       <dt>
         Issuer Service
       </dt>
       <dd>
-        The <a>entity</a> that creates a <a>claim</a> and associates it with a
-        particular <a>subject</a>.
+        The <a>entity</a> that creates generates and delivers a <a>Verifiable Credential</a> to the Issuer Coordinator
+        which contains a some set of <a>claims</a> made by the Issuer Coordinator about one or more subjects.
```
Comment (TallTed) [inline index.html]:
```suggestion
        The <a>entity</a> that creates, generates, and delivers a <a>Verifiable Credential</a> — which contains
        a some set of <a>claims</a> made by the Issuer Coordinator about one or more subjects — to the Issuer Coordinator. 
```

**Comment by eric-schuh** [comment]
> 

On the respec errors, I believe most of the terminology errors are due to the PR system. The same terminology section in the main repo properly grabs the terms and the majority of red underlines are not seen. I think I would like to just get this merged and see what set of respec issues still exist then fix those in a new PR since currently it is hard to say which are from the PR system and which are real (at least to me)

File: index.html [hunk 21]
```diff
@@ -152,45 +152,758 @@
...
+        <a>Issuer Service</a>, <a>Issuer Coordinator</a>
+        <a>Verifier Service</a>, and <a>Verifier Coordinator</a>. 
+      </p>
+
+    <dl>
+      <dt>
+        Issuer Service
+      </dt>
+      <dd>
+        The <a>entity</a> that creates, generates, and delivers a <a>Verifiable Credential</a> — which contains
```
Comment (jandrieu) [inline index.html]:
```suggestion
        The <a>component</a> that creates, generates, and delivers a <a>Verifiable Credential</a> — which contains
```

File: index.html [hunk 22]
```diff
@@ -152,45 +152,758 @@
...
+        a some set of <a>claims</a> made by the Issuer Coordinator about one or more subjects — to the Issuer Coordinator. 
+      </dd>
+      <dt>
+        Issuer Coordinator
+      </dt>
+      <dd>
+        The <a>entity</a> that deals with business rules and policies around a
```
Comment (jandrieu) [inline index.html]:
```suggestion
        The <a>component</a> that deals with business rules and policies around a
```

File: index.html [hunk 23]
```diff
@@ -152,45 +152,758 @@
...
+        user's request for a particlar <a>claim</a> to be issued.
+      </dd>
+      <dt>
+        Verifier Service
+      </dt>
+      <dd>
+        The <a>entity</a> performing the verification of a <a>Verifiable Credential</a> containing <a>claims</a> about a given <a>subject</a>.
```
Comment (jandrieu) [inline index.html]:
```suggestion
        The <a>component</a> performing the verification of a <a>Verifiable Credential</a> containing <a>claims</a> about a given <a>subject</a>.
```

File: index.html [hunk 24]
```diff
@@ -152,45 +152,758 @@
...
+      </dd>
+      <dt>
+        Verifier Coordinator
+      </dt>
+      <dd>
+        The <a>entity</a> that deals with business rules and policies around a
```
Comment (jandrieu) [inline index.html]:
```suggestion
        The <a>component</a> that deals with business rules and policies around a
```

File: index.html [hunk 25]
```diff
@@ -152,45 +152,758 @@
...
+        user's request for a particular <a>Verifiable Credential</a> to be verified. 
+      </dd>
+      <dt>
+        Subject
+      </dt>
+      <dd>
+        The <a>entity</a> about whom a <a>claim</a> is issued.
```
Comment (jandrieu) [inline index.html]:
```suggestion
        The <a>entity</a> about whom a <a>claim</a> is made.
```

File: index.html [hunk 26]
```diff
@@ -152,45 +152,758 @@
...
+        </dd>
+        <dt>
+          <udef>E.3 Transferring schools</udef>
+        </dt>
+        <dd>
+          Rocky is an undergraduate student at Wossamotta U.  His school provides a
+          <a>credential repository</a> service to all students and alumni, so he chooses
+          to use it. In his third year, Rocky decides to transfer to Moosylvania Tech.
+          They do not offer a service, but he does not want to continue to use the
+          service of his old (and now rival school) so he moves his <a>claims</a> to the
```
Comment (jandrieu) [inline index.html]:
```suggestion
          service of his old (and now rival school) so he moves his <a>verfifiable credentials</a> to the
```

File: index.html [hunk 27]
```diff
@@ -152,45 +152,758 @@
...
+      <dl class="left dl-horizontal">
+        <dt>
+          <udef>R.1 Address verification</udef>
+        </dt>
+        <dd>
+          Francis has found the perfect pair of shoes. When processing orders, Giant Shoe
+          Company wants to be certain that his shipping address is accurate (inaccurate
+          addresses are very expensive in terms of customer service). They offer a
+          discount for customers who make verifiable addresses available as part of the
+          checkout process. Francis offers his certificate to Giant Shoe Company's
```
Comment (jandrieu) [inline index.html]:
```suggestion
          checkout process. Francis offers a verifiable credential--containing a claim by a known authority that a specific address has been verified--to Giant Shoe Company's 
```

File: index.html [hunk 28]
```diff
@@ -152,45 +152,758 @@
...
+          verifier coordinator, which verifies Francis's address, making use of 
+          Generic Verfifier Service Inc. Francis gets the perfect shoes for
+          even less than he expected.
+        </dd>
+        <dt>
+          <udef>R.2 Adult beverages</udef>
+        </dt>
+        <dd>
+          June goes to her local beer and wine store to buy a bottle of wine. She submits
+          her identity credential that lets the liquor store owner know that she is over
```
Comment (jandrieu) [inline index.html]:
```suggestion
          a proof-of-age credential that lets the liquor store owner know that she is over
```

File: index.html [hunk 29]
```diff
@@ -152,45 +152,758 @@
...
+        </dt>
+        <dd>
+          On a bright Sunday, Oskar remembers that he still needs to buy his wife a
+          precious gift for their wedding anniversary. However, he is acutely aware that
+          it is precisely in weekends that gangs set up fraudulent web shops that claim
+          to sell such gifts, while in fact they only take the cash, and disappear on
+          Mondays. So before actually purchasing a gift from the web shop of his choice,
+          he requests the shop to provide a credential issued by the chamber of commerce,
+          that contains proof of legitimacy. After having verified that the shop is
+          legit, by making use of the Wallet Verifier Service built into his
```
Comment (jandrieu) [inline index.html]:
```suggestion
          legit, by making use of the Verifier Service built into his
```

File: index.html [hunk 30]
```diff
@@ -152,45 +152,758 @@
...
+          <udef>F.1 Reuse know your customer</udef>
+        </dt>
+        <dd>
+          Jane is opening an account at MidBank in Finland.  As part of that process,
+          the bank asks her to provide two from a variety of possible sources to confirm
+          her identity &mdash; a so-called &quot;Know Your Customer&quot; check. She goes
+          to the bank website, logs in, and navigates to the KYC page. She then
+          selects the government-supplied <a>verifiable credentials</a> that confirm she
+          receives postal mail at a certain address and that she has a national ID card,
+          submitting these. MidBank has stood up their own verification service, which
```
Comment (jandrieu) [inline index.html]:
```suggestion
          receives postal mail at a certain address and that she has a national ID card.
          She submits these to Midbank's verification coordinator. MidBank has their own verification service, which 
```

File: index.html [hunk 31]
```diff
@@ -152,45 +152,758 @@
...
+          BigBank into the repository at WallStreetCo to experiment with their service,
+          but continues to use the service from BigBank while she is testing.
+        </dd>
+        <dt>
+          <udef>F.5 New bank account from home</udef>
+        </dt>
+        <dd>
+          Alice wants to open a new bank account. BigOnlineBank offers the ability to
+          do this from home if she can provide electronic credentials.  She offers
+          government-issued certificates that verify her identity (address, national
```
Comment (jandrieu) [inline index.html]:
```suggestion
          government-issued certificates that verify her attributes (address, national
```

File: index.html [hunk 32]
```diff
@@ -152,45 +152,758 @@
...
+      <dl class="left dl-horizontal">
+        <dt>
+          <udef>H.1 Prescribing</udef>
+        </dt>
+        <dd>
+          Barney is a physician, and has recently become board certified in his state.
+          The state's board makes use of the international Medical Certificate Issuer
+          to issue credentials that adhere to the international standards for 
+          medical practitioners. Upon completing his state's board certificate test, 
+          Barney is issued a digital certificate confirming that he is
```
Comment (jandrieu) [inline index.html]:
```suggestion
          Barney is issued a verifiable credential confirming that he is
```

File: index.html [hunk 33]
```diff
@@ -152,45 +152,758 @@
...
+          certified to practice medicine in that state.  Barney can now use this
+          certificate when writing prescriptions and referrals, thereby improving
```
Comment (jandrieu) [inline index.html]:
```suggestion
          credential when writing prescriptions and referrals, thereby improving
```

File: index.html [hunk 34]
```diff
@@ -152,45 +152,758 @@
...
+        </dd>
+        <dt>
+          <udef>H.3 Insurance claim</udef>
+        </dt>
+        <dd>
+          Tracy has a sore throat soon after moving to a new town. She finds a physician
+          through her health care network and goes in for treatment. She is a new
+          patient, so the clinic needs to know who she is and how she will be paying.
+          When checking in, she presents her <a>verifiable credential</a> that
+          demonstrates her identity and her proof of insurance, which is checked by
```
Comment (jandrieu) [inline index.html]:
```suggestion
          demonstrates her identity attributes and her proof of insurance, which is checked by
```

File: index.html [hunk 35]
```diff
@@ -152,45 +152,758 @@
...
+          by the Generic Verifier Service that the cruise ship uses, and sends Barney
+          a response email thanking him for the assistance. 
+        </dd>
+        <dt>
+          <udef>C.3 Bad university</udef>
+        </dt>
+        <dd>
+          Jane was issued a certificate by BigTraining Co., indicating that she was a
+          trained Project Manager.  It was later discovered that BigTraining Co. was not
+          actually training anyone, and their organization's certificate was revoked via
```
Comment (jandrieu) [inline index.html]:
```suggestion
          actually training anyone, and their organization's authority to issue Project Manager certifications was revoked via
```

File: index.html [hunk 36]
```diff
@@ -152,45 +152,758 @@
...
+          the US Department of Education's Accreditation Database. Jane's credential is
+          therefore invalid, and when Jane goes to submit the credentail received from
```
Comment (jandrieu) [inline index.html]:
```suggestion
          therefore invalid, and when Jane goes to submit the credential received from
```

File: index.html [hunk 37]
```diff
@@ -152,45 +152,758 @@
...
+        </dd>
+        <dt>
+          <udef>C.5 Social authority</udef>
+        </dt>
+        <dd>
+          Josie is a healthcare worker that has created a profile on a professional
+          social network to make herself readily available for new opportunities in the
+          workforce. She lists her employment history and credentials including degrees,
+          certificates, and digital badges. The website requests verification of her
+          credential <a>claims</a> in order for her credentials to be visible when she
```
Comment (jandrieu) [inline index.html]:
```suggestion
          <a>claims</a> in order for her credentials to be visible when she
```

**Review by jandrieu — CHANGES_REQUESTED** [review]
Overall, this is a great update. I've suggested some clean up. We also might think through how to be more rigorous in our use of credentials v certifications and certificates. There's a lot of confusion about how a real-world certification is represented in a VC and it would be good to tease out language to delineate in the use cases when it is the VC that we are talking about verses the underlying certification.

**Comment by eric-schuh** [comment]
> Overall, this is a great update. I've suggested some clean up. We also might think through how to be more rigorous in our use of credentials v certifications and certificates. There's a lot of confusion about how a real-world certification is represented in a VC and it would be good to tease out language to delineate in the use cases when it is the VC that we are talking about verses the underlying certification.

Added this note as an issue for next chunk of work. Merging this PR after accepting the general text changes made along with this review

**Pull request merged**